packages/cli/src/commands/typescript/generate/generate.ts

```diff
@@ -67,12 +67,17 @@ export const command: CommandModule<
             default: "commonjs",
             choices: ["commonjs", "module"],
           },
+          version: {
+            type: "string",
+            description: "Version of the generated code, or 'dev'",
+            demandOption: true,
+          },
```

Comment:
Do we want to restrict this to only valid semver strings or something?

Reply:
Hmm, do you know what foundry-sdk-generator CLI does? In Dev Console the server side SDK versions are validated before the CLI gets called I'm not sure if the CLI itself also validates. 

How valuable do you think this is for the intended usage of this CLI? I'm not too familiar with the plans on the 2.0 CLI which only generates the files but not the full NPM package with a name/version